ironfish-cli/src/utils/allocations.ts

```diff
@@ -0,0 +1,61 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
```

Comment:
No source change, just moved for reusage.

Reply:
This is also used in the `genesisadd` command if you wanted to delete from there as well, i'm not too concerned about it though